pkg/tfbridge/info.go

```diff
@@ -133,6 +133,12 @@ type ProviderInfo struct {
 	//
 	// See https://github.com/pulumi/pulumi-terraform-bridge/issues/1501
 	XSkipDetailedDiffForChanges bool
+
+	// Enables generation of a trimmed, runtime-only metadata file
+	// to help reduce resource plugin start time
+	//
+	// See also pulumi/pulumi-terraform-bridge#1524
+	GenerateRuntimeMetadata bool
```

Comment:
Is there any reason why a provider would not want this flag enabled? 

Put another way, if we were 100% confident that this would work as intended, would we enable it for all of our providers?

Reply:
There's not a lot of value for smaller providers, the data in the runtime config is strictly redundant with bridge-metadata.json (while being harder to read/edite) and it requires changing up the init a bit to use the runtime file for the runtime binary. So for now, I don't think it's worthwhile to try to enable this everywhere, lots of little frictions for minimal benefit on most providers.

We might revisit this as part of a larger effort to distinguish gentime/runtime though.

Reply:
That makes sense to me.